src/core/mod.rs

```diff
@@ -142,6 +146,18 @@ impl BPFBuilder {
         self
     }
 
+    /// Sets whether or not to ignore 
```

Comment:
Can you expand on this comment?

Reply:
Ah, woops.  This is totally an incomplete comment that I missed.  Will fix.